raspberry_pi_installation.md

```diff
@@ -177,30 +170,35 @@ and it will show something like
 
 as seen on the last row the interface is attached to `ttyUSB0` so the device path will be `/dev/ttyUSB0`
 
-To set it up as input for the SignalK server go to the admin UI and ”Server => Data Providers”.
+To set it up as input for the SignalK server go to the admin UI and ”Server => Connections”.
 Add a provider with settings according to the picture and click on ”Apply”
 
-![provider_nmea2000](https://user-images.githubusercontent.com/16189982/43796707-51a88a9e-9a85-11e8-8253-3fb20a9ba3a5.jpeg)
+![SK_N2](https://user-images.githubusercontent.com/16189982/69761979-3d8b5080-1169-11ea-83a2-b688fd1178b3.png)
 
 The SignalK server **must be restarted** to accept the changes so click on "Restart" in the upper right corner.
 
-
 After the restart check the data with the Webapp Instrumentpanel in the admin UI.
 
 NMEA0183
 
 If You have a NMEA0183 network, plug in Your NMEA0183 to USB interface and do the same procedure as for the Actisense above.
 With a configuration maybe looking like this
 
-![provider_nmea0183](https://user-images.githubusercontent.com/16189982/36218419-93f81c84-11b4-11e8-86c2-8319b9013b90.jpeg)
+![SK_N1](https://user-images.githubusercontent.com/16189982/69761986-44b25e80-1169-11ea-9401-2c06ace9eb3b.png)
 
 ## Sample files
 
-If You don’t have any NMEA interface hardware You could set up a provider with the sample files instead.
+If You don’t have any NMEA interface hardware You could set up a "File Stream" connection with the sample files instead.
```

Comment:
We already have --sample-nmea0183-data documented here, maybe we could just remove this Sample files section?

Reply:
Njaa......
Earlier in the text You are advised to use `signalk-server --sample-nmea0183-data`, to check the installation, that's true, but after You have used the `sudo signalk-server-setup` it's not that easy for a newbie to run the test again. This is due to that You have to stop the 2 deamons before using it again, and then, after the test, restart. Also if You want to use the N2K data instead, to test `canboat`, there is no info available. Another thought is that this info also could help OpenPlottter users finding test files.